README.md

```diff
@@ -8,6 +8,7 @@
 [![Latest version on conda-forge](https://img.shields.io/conda/v/conda-forge/pygmt?style=flat-square)](https://anaconda.org/conda-forge/pygmt)
 [![GitHub Actions Tests status](https://github.com/GenericMappingTools/pygmt/workflows/Tests/badge.svg)](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests.yaml)
 [![GitHub Actions GMT Dev Tests status](https://github.com/GenericMappingTools/pygmt/workflows/GMT%20Dev%20Tests/badge.svg)](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests_dev.yaml)
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/GenericMappingTools/pygmt/main.svg)](https://results.pre-commit.ci/latest/github/GenericMappingTools/pygmt/main)
```

Comment:
I feel this badge is not necessary.

Reply:
Would it be more useful if it links to https://results.pre-commit.ci/repo/github/85352251 instead?

Reply:
pre-commit CI is unlikely to fail after this PR is merged. So I guess the badge and the status report is not necessary.

Reply:
Technically, `pre-commit run` would only run on changed files by default (unless you use the `--all files` flag), so there is a chance that the `pre-commit` CI check will pass on PRs, but fail on the main branch if some lint rules have changed on other files. In that case, it might be good to have this badge to see what the error is (or we can click on the red cross next to the commit msg if logged into GitHub).

Reply:
The CI status badge can be removed after we run `pre-commit` in the "Style Checks" workflow.

Reply:
Removed at d937cacf0d43ba76749ef6b6476a75b38bfd685d